wpilibc/src/test/native/c/test.c

```diff
@@ -1,5 +1,5 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
```

Comment:
This file didn't actually change?  May need a rebase/squash so wpiformat doesn't complain?